pkg/crd/generator/testData/pkg/apis/fun/v1alpha1/toy_types.go

```diff
@@ -74,11 +74,20 @@ type ToySpec struct {
 	Location map[string]string `json:"location"`
 }
 
+// Inter interface.
+type Inter interface{}
```

Comment:
Why not `Interface` as name?

Reply:
Yeah, even if I change the name to `Interface`, the o/p remains same.